purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -369,17 +371,46 @@ class Purchases internal constructor(
                     override fun onReceived(inappStoreProducts: List<StoreProduct>) {
                         callback.onReceived(subStoreProducts + inappStoreProducts)
                     }
+
                     override fun onError(error: PurchasesError) {
                         callback.onError(error)
                     }
                 })
             }
+
             override fun onError(error: PurchasesError) {
                 callback.onError(error)
             }
         })
     }
 
+    /**
+     * TODO BC5 javadocs
+     */
+    fun purchase(
+        purchaseConfig: Purchase,
+        listener: NewPurchaseCallback
+    ) {
```

Comment:
Love it

Reply:
This is going to be so much nicer to maintain 💪